custom_components/waste_collection_schedule/waste_collection_schedule/source/stockton_gov_uk.py

```diff
@@ -0,0 +1,104 @@
+from datetime import datetime
+
+import requests
+import re
+import json
+import base64
+from bs4 import BeautifulSoup
+from waste_collection_schedule import Collection
+
+# Source code based on gateshead_gov_uk
+
+TITLE = "Stockton-on-Tees Borough Council"
+DESCRIPTION = "Source for Stockton-on-Tees Borough Council."
+URL = "https://www.stockton.gov.uk/"
+TEST_CASES = {
+    "100110203615": {"uprn": "100110203615"},
```

Comment:
As we had this issue multiple times now: Could you add a test case with an integer (instead of string)?